src/main/java/com/bokov/FirstTGBot/repositores/BTCRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.bokov.FirstTGBot.repositores;
+
+import com.bokov.FirstTGBot.models.BTC;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+
+import java.util.Date;
+import java.util.List;
+
+@Repository
+public interface BTCRepository extends JpaRepository<BTC, Integer> {
+    List<BTC> findAllByDateOfRequestBetween(Date date, Date date2);
```

Comment:
method name is not clear for me. I'd suggest using Optional<BTC> and pass single date to parameters.

In this case:
if today -> you will pass just DateTime.now()
for yesterday -> you will pass just DateTime.now().munisDays(1);